resources/definitions/fdmprinter.def.json

```
@@ -5767,7 +5766,7 @@
                     "description": "If brim is only on outside then parts fully enclosed inside another part will get a brim which might overlap with the internal holes of the outer part. This setting controls how far to stay away from those internal holes. Set to a high value to prevent any brim from being generated for parts enclosed within the holes of other parts.",
                     "unit": "mm",
                     "type": "float",
-                    "default_value": 5,
+                    "value": "2.5",
```

```suggestion
                    "default_value": "2.5",
```
A setting must always have a default value. The engine can't use this folder otherwise.

---

both value and default value are set to "2.5" should I only choose one of the two?